### DISCUSSION TITLE
Eslint with `next/core-web-vitals` cannot work properly in my VSCode

### DISCUSSION THREAD

**Discussion by WendaoLee** [vercel/next.js#62879] [Help]
### Summary

I'm collaborating on a Next.js project with a friend. While my friend encounters eslint errors in their VSCode setup, I don't experience the same issue. Upon investigation, I discovered that the log (shown below) indicates that the core-web-vitals extension failed to load properly in my environment. How can I address this issue?
I would be appreciated for your help.

### Additional information

```js
2024-03-05T14:23:54.216Z eslintrc:config-array-factory Loading {extends:"next/core-web-vitals"} relative to G:\Github\project-next\my-app\.eslintrc.json
2024-03-05T14:23:54.223Z eslintrc:config-array-factory package.json was not found: Cannot find module 'eslint-config-next/core-web-vitals/package.json'
```

**Comment by WendaoLee** [comment]
BTW,I have tried re-install all the dependencies and even I tried to create new project,the same log still happened.

**Comment by WendaoLee** [comment]
It's not a problem.Even I don't know why,this doesn't make any sense.My trouble caused by a noob setting.Holy shit,how fucked I am.

**Comment by mezzouhri** [comment]
Hi, did you fix this problem, please ?
I have the same error

**Comment by HovhannesKeshishyan** [comment]
hi, does version of "eslint-config-next" match the version of "next" ?
